content/sensu-go/5.16/guides/using-check-aggregates.md

```diff
@@ -0,0 +1,285 @@
+---
+title: ""
+linkTitle: "Using Check Aggregates"
+description: ""
+weight: 350
+version: "5.16"
+product: "Sensu Go"
+platformContent: false
+menu:
+  sensu-go-5.16:
+    parent: guides
+---
+
+Sensu provides the capability to monitor groups of checks or entities via aggregates. In Sensu Go, this is done through the use of labels. In this tutorial, we'll walk through configuring both an entity aggregate and a check aggregate.
+
+# Entity Aggregate
+
+Let's start with configuring an entity.
+
+The first thing we'll need to do is add a label to our agent.
+
+```yaml
+---
+# Sensu agent configuration
+
+##
+# agent overview
+##
+
+labels:
+  server_type: "webservers"
+
+```
+
+In this example, we've added the label `server_type` with the value `"webservers"`--it's useful to know if a number of agents on our webservers stop reporting in rather than just a single node. Let's add a similar label to a check.
+
+# Check Aggregate
+
+The check we'll use for our aggregate is an http metric check for a ficticious app site 
+
+```yaml
+---
+type: CheckConfig
+api_version: core/v2
+metadata:
+  name: cpu-check
+  namespace: default
+  labels:
+    app_group: "identity_servers"
+spec:
+  command: "check-http.rb -u https://identity.{{ .entity.name }}.example.com"
+  interval: 10
+  publish: true
+  handlers:
+  - slack
+  runtime_assets:
+  - sensu-plugins/sensu-plugins-http
+  - sensu/sensu-ruby-runtime
+  subscriptions:
+  - linux
+
+```
+
+While Sensu is capable of monitoring singular entities, it is also capable of monitoring groups of entities (and checks) through the use of aggregates. _______________ . Let's get started.
+
+===
+
+## Entity Aggregates
+
+In order for a check or entity to be considered as part of an aggregate it must have a label, or set of labels assigned to it. We'll start by setting up a fictional scenario in which we have 20 webservers serving a number of applications. In this scenario, we might not care if a singular server stops responding, but we might care if say, 15 of the 20 stop responding. 
+
+To make an entity part of an aggregate, we would need to add a label in our `/etc/sensu/agent.yml`. For example:
+
+```yaml
+---
+# Sensu agent configuration
+
+##
+# agent overview
+##
+name: "webserver01.example.com"
+namespace: "default"
+subscriptions:
+  - webservers
+labels:
+  server_role: "webserver"
+```
+
+After adding the label, we'll make sure to restart our agent to pick up the change in configuration:
+
+`systemctl restart sensu-agent`
+
+We'll need to set up a check that will look at events with the label we've assigned to the entity and ensure that these events are in an OK status. Let's look at an example check:
+
```

Comment:
Could it be confusing that we have not explained how events are part of this? We have explained that you need to add a label to an entity and mentioned that readers will eventually need to set up a check aggregate, but we haven't explained something like "The events this entity produces will be associated with the `webserver` label, which will allow us to watch for those events with a check."